ctk/common/src/test/kotlin/org/codice/compilance/verification/core/requests/CoreLogoutRequestProtocolVerifierSpec.kt

```diff
@@ -23,6 +27,8 @@ import java.time.Instant
 import java.util.UUID
 
 class CoreLogoutRequestProtocolVerifierSpec : StringSpec() {
+    override fun listeners(): List<TestListener> = listOf(ReportListener)
```

Comment:
❓ I keep thinking there should be a better way of doing this instead of overriding the function on every one of our reportable tests, but short of inserting a new base class between `StringSpec` and all of ours, I really can't. And that seems like overkill.

Reply:
Yeah I tried looking for a better way and other than having a superclass there isn't something I can do